hack/update-deps.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Identifies go_library() bazel targets in the workspace with no dependencies.
+# Deletes the target and its files. Cleans up any :all-srcs references
+
+# By default just checks for extra libraries, run with --fix to make changes.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+# Remove //vendor/golang.org/x/text/internal:go_default_library deps
+#
+# Unused text/gen.go file imports text/internal, which prevents dep prune from
```

Comment:
I feel like we should also `rm` this file since it won't build.

Reply:
Once we remove the dep and bazel runs then `hack/prune-libraries.sh` removes it, yes